src/engraving/libmscore/ambitus.cpp

```diff
@@ -162,7 +157,7 @@ void Ambitus::setTrack(track_idx_t t)
 void Ambitus::setTopPitch(int val, bool applyLogic)
 {
     if (!applyLogic) {
-        _topPitch   = val;
+        m_topPitch   = val;
```

Comment:
When you're doing things with code style anyway, it might be nice to do something about these awkward spaces before '=' signs too. But that can be done in a separate PR.